runtime/src/iree/hal/drivers/hip/graph_command_buffer.c

```diff
@@ -11,11 +11,13 @@
 #include <stdint.h>
 
 #include "iree/base/api.h"
+#include "iree/base/attributes.h"
```

Comment:
This is included as part of `base/api.h` so no need to pull in specifically

Reply:
done